src/watcher.js

```diff
@@ -30,6 +31,8 @@ const eventContract = new web3Instance.eth.Contract(config.eventAbi, config.even
 const lastBlockRedisKey = `${config.id}:lastProcessedBlock`
 let lastProcessedBlock = BN.max(config.startBlock.sub(ONE), ZERO)
 
+const maxProcessingTime = process.env.MAX_PROCESSING_TIME
```

Comment:
We need to define a default value which will be used if `MAX_PROCESSING_TIME` was missed by some reason in the configuration file.